### PR TITLE
Update recordTrimEdit.py (test #11)

### PR DESCRIPTION
**Pull request by Afonsobrasil**
# Error solving( test warning )
This is in test, so this dont work
Also, soon if it rlly dont work, i'll update it.
this is made due to the issue 11.

### CODE REVIEW

**Comment by Afonsobrasil** [comment]
i added a if to looks great.

File: recordTrimEdit.py [hunk 1]
```diff
@@ -5,14 +5,19 @@
...
+from pygame_widgets.dropdown import Dropdown
 from pygame._sdl2 import (
     get_audio_device_names,
     AudioDevice,
     AUDIO_S16,
     AUDIO_ALLOW_FORMAT_CHANGE,
 )
-import pygame_widgets
-from pygame_widgets.dropdown import Dropdown
+try:
```
Comment (ben9583) [inline recordTrimEdit.py:16]: Is there any particular reason this would fail?

File: recordTrimEdit.py [hunk 2]
```diff
@@ -21,24 +26,49 @@
 default_margin = 0.25
 default_threshold = 200
 default_mic_index = 0
-
+
```
Comment (ben9583) [inline recordTrimEdit.py:29]: formatting

**Review by ben9583 — CHANGES_REQUESTED** [review]
This is a cool feature! There are a handful of issues here and I get the sense you haven't run this before (line 36 is a syntax error). Please test this yourself after fixing the errors and get back to me here

**Comment by Afonsobrasil** [comment]
Ok, i'll test it.

**Comment by Afonsobrasil** [comment]
It is really taking more than i thinked. Hope i fix it.

**Comment by Afonsobrasil** [comment]
I'll commit it and you test the code. I think that is my mic is really bad.

**Comment by Afonsobrasil** [comment]
FINISHED! Error #6 is not letting it be  100% complete, but accept this, that i'll fix it.

**Comment by Afonsobrasil** [comment]
Also there was an error in recordTrimEdit that dont let it open.
i fixed it lol

**Review by Afonsobrasil — COMMENTED** [review]
All done

**Comment by ben9583** [comment]
I think whatever it is you're doing with the `try...except` block on the import and the changes to config loading is beyond the scope of this issue, so I'll be removing it for now but feel free to create a separate PR for those changes

**Pull request closed without merging**